presto-iceberg/pom.xml

```diff
@@ -10,10 +10,18 @@
     <description>Presto - Iceberg Connector</description>
     <packaging>presto-plugin</packaging>
 
+    <repositories>
+        <repository>
+            <id>iceberg-release-candidate</id>
+            <name>Iceberg Release Candidate</name>
+            <url>https://repository.apache.org/content/repositories/orgapacheiceberg-1121/</url>
+        </repository>
+    </repositories>
+
```

Comment:
Would you mind explaining a bit more about the purpose of this part?

Reply:
@ChunxuTang this is only temporary so that we can fetch the RC version of the library. It will be removed once the official version is out

Reply:
Gotcha! Thanks for the clarification.